README.md

```diff
@@ -83,7 +83,7 @@ for an **Advanced** example, check out:
 | **`children`**              | `undefined`                                                                      | `Function`                           | Render prop component. See [docs below](#render-props) for the options passed                                                                                                       |
 | `initialDate`, `date`       | `new Date()`                                                                     | `Date`, `String`, `Number`, `Moment` | `initialDate` sets the initial state of `date` for uncontrolled usage, otherwise use `date` for controlled usage. Used as the date to center the calendar around                    |
 | `initialNumDays`, `numDays` | `35`                                                                             | `Number`                             | Number of days the calendar should display. If `numDays` > 10, this will be raised to the next multiple of 7. Use `initialNumDays` for uncontrolled usage, `numDays` for controlled |
-| `events`                    | `[]`                                                                             | `Array<{ date: DateLike }>`          | Events passed into the calendar. These objects will be injected into the correct array by date                                                                                      |
+| `events`                    | `[]`                                                                             | `Array<{ date: Moment, date2: Moment }>`  | Events passed into the calendar. These objects will be injected into the correct array by date. `date2` is optional and only valid for events spanning a date range. `date` can also be specified as `date1`.                                                                           |
```

Comment:
I believe DateLike wasn't fully functioning here as was: if a user passed a string, it would bork at `getDateKey` when we called `toISOString` on a string, for example. For now, I just made this be a moment date but it wouldn't be too challenging to just validate before using whether it is a moment date and if not to attempt to wrap it. happy to add that as well.

Reply:
I don't want the docs to explicitly say Moment here, since I want people to be able to use CalendarX _without_ `moment`. If something is failing when using strings then we should change that thing instead of the documentation 🙂